packages/virtual-core/src/index.ts

```diff
@@ -156,7 +156,10 @@ export const observeElementOffset = <T extends Element>(
       )
 
   const createHandler = (isScrolling: boolean) => () => {
-    offset = element[instance.options.horizontal ? 'scrollLeft' : 'scrollTop']
+    const { horizontal, isRtl } = instance.options
+    offset = horizontal
+      ? element['scrollLeft'] * (isRtl && -1 || 1)
```

Comment:
Let's keep it simple 
```suggestion
      ? element['scrollLeft'] * (isRtl ? -1 : 1)
```

Did you test it on all modern browsers?

Reply:
We are used to adhering to ESlint's rules regarding ternaries: https://eslint.org/docs/latest/rules/no-nested-ternary

The scenario in the included videos was tested in Safari, Chrome, Firefox, Brave, Opera, and Edge.